{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -19,7 +20,7 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
 ]
 
-dependencies = ["numpy==1.25.2"]
```

Comment:
This version of numpy is incompatible with python 3.8

Reply:
Probably better to remove the numpy dependency altogether and just have an empty list?

Reply:
Sounds good!